src/build/build.ts

```diff
@@ -82,66 +82,45 @@ export async function build(
           .pipe(gulpif(/\.html$/, new HTMLOptimizeStream(optimizeOptions.html)))
           .pipe(polymerProject.rejoinHtml());
 
-  let buildStream = mergeStream(sourcesStream, depsStream);
-
-  let unbundledPhase = forkStream(buildStream)
-                           .once(
-                               'data',
-                               () => {
-                                 logger.info('Generating build/unbundled...');
-                               })
-                           .pipe(gulpif(
-                               options.insertDependencyLinks || false,
-                               new PrefetchTransform(polymerProject)))
-                           .pipe(dest(unbundledBuildDirectory));
-
-  let bundledPhase = forkStream(buildStream)
-                         .once(
-                             'data',
-                             () => {
-                               logger.info('Generating build/bundled...');
-                             })
-                         .pipe(polymerProject.bundler)
-                         .pipe(dest(bundledBuildDirectory));
+  let buildStream =
+      <NodeJS.ReadableStream>mergeStream(sourcesStream, depsStream)
+          .once('data', () => {
+            logger.debug('Analyzing build dependencies...');
+          });
+
+  if (options.bundle) {
+    buildStream = buildStream.pipe(polymerProject.bundler);
+  } else {
+    buildStream = buildStream.pipe(gulpif(
+        options.insertDependencyLinks || false,
+        new PrefetchTransform(polymerProject)));
+  }
+
+  buildStream = buildStream
+                    .once(
```

Comment:
What do you have in mind?

I fiddled around with clang-format on this line a bit, but had no luck. The same thing happens if I use `function() {` instead of `() => {`.